ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -397,13 +365,18 @@ else
 
 		sensors start
 		commander start
+
+		if param greater -s DSHOT_CONFIG 0
+		then
+			dshot start
+		fi
```

Comment:
`DSHOT_CONFIG` should be removed and we can start `dshot` unconditionally. It will exit if none of the groups are configured as dshot.